amendments to daml finance settlement docs (#526)
digital-asset/docs.daml.com: docs/2.8.0/docs/daml-finance/tutorials/advanced-topics/lifecycling/intermediated-lifecycling.rst

=================================================

The parties who sign the
:ref:`Batch <type-daml-finance-interface-settlement-batch-batch-97497>` contract (the instructor
and consenters) can exercise the ``Cancel`` choice of the
:ref:`Batch <type-daml-finance-interface-settlement-batch-batch-97497>` to cancel all associated
:ref:`Instructions <type-daml-finance-interface-settlement-instruction-instruction-30569>`
atomically.

> Settlement can be obstructed if the responsible parties do not carry out their duties to allocate
> and approve the settlement instructions, as the success of a batch hinges on these actions being
> completed.

Reviewer: ```suggestion
Settlement can also be obstructed if the responsible parties do not fully allocate
and approve the settlement instructions.

```

Reviewer: Note: I don't think the line about the "success of a batch" adds anything, so I deleted it. Feel free to veto that change and I'll try again.

And I added "also" to provide a connection between the two paragraphs, which was a little clearer before these edits (though I see why the revisions are needed).
